vera.galstyan/repo/Lesson_3/exercise_03/exercise_03.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+#1
+print("*.*.*.*.*.*","\n*.*.*.*.*.*","\n*.*.*.*.*.*","\n*.*.*.*.*.*","\n*.*.*.*.*.*","\n*.*.*.*.*.*")
```

Comment:
RE: This line doesn't print the considered shape. Please run locally and be sure that the shape is 100% similar to the shape in the exercise.